src/LM/preprocessing.jl

```diff
@@ -0,0 +1,90 @@
+"""
+Return all possible ngrams generated from sequence of items, as an Array{String,1}
+# Example
+
+>>>seq=["To","be","or","not"]
+>>>a = everygram(seq,min_len = 1, max_len = -1)
+   10-element Array{Any,1}:
+ "or"          
+ "not"         
+ "To"          
+ "be"                 
+ "or not" 
+ "be or"       
+ "be or not"   
+ "To be or"    
+ "To be or not"
+   
+"""
+function everygram(seq; min_len::Int=1, max_len::Int=-1)
+     ngram = []
+    if max_len == -1
+        max_len = length(seq)
+    end
+    for n in range(min_len, stop =max_len)
+       temp = keys(TextAnalysis.ngramize(TextAnalysis.Languages.English(),seq,n))
+       ngram = append!(ngram,temp)
+   end
+   return(ngram)
+end
+
+"""
+   padding _ngram is used to pad both left and right of sentence and out putting ngrmas
+   
+   It also pad the original input Array of string 
+# Example Usage
+>>>example = ["1","2","3","4","5"]
+      
+>>> example = ["1","2","3","4","5"]
+>>> padding_ngram(example ,2 , pad_left=true,pad_right =true)
+    5-element Array{String,1}:
+ "1"
+ "2"
+ "3"
+ "4"
+ "5"
+"""
+function padding_ngram(word,n =1 ;pad_left=false,pad_right=false ,left_pad_symbol="<s>", right_pad_symbol ="</s>")
```

Comment:
Documentation are good, but the syntax seems a bit off. Go through the julia docs on documentation: https://docs.julialang.org/en/v1/manual/documentation/index.html

See this for example docstrings: [FileDocument](https://github.com/JuliaText/TextAnalysis.jl/blob/a959025daa6908cbff2c0efc4aeed5c5bbecc479/src/document.jl#L48)